pkg/helm/version_detect.go

```diff
@@ -28,6 +31,11 @@ const (
 	Helm3Lib = ClientType("Helm3Lib")
 )
 
+func checkPostRenderer(path string) error {
+	_, err := os.Stat(path)
```

Comment:
it's ok to return raw OS error to user here?

Reply:
yes, I don't care much about a deprecated way of deploying releases. All I want is to make sure  the post-renderer binary is in place.